fs/vfs/main.cc

```diff
@@ -1245,6 +1245,39 @@ int link(const char *oldpath, const char *newpath)
     return -1;
 }
 
+OSV_LIBC_API
+int linkat(int olddirfd, const char *oldpath, int newdirfd, const char *newpath, int flags)
+{
+    if (flags & AT_SYMLINK_FOLLOW) {
```

Comment:
Maybe add `UNIMPLEMENTED("linkat() with AT_SYMLINK_NOFOLLOW");`
like we have with accessat(), so in the future if we have an application that needs it, it will be more visible?

Reply:
Added `WARN_ONCE` in the latest update.